api/NetworkInformation.json

```
@@ -457,6 +409,56 @@
           }
         }
       },
+      "typechange_event": {
+        "__compat": {
+          "description": "<code>typechange</code> event",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/NetworkInformation/typechange_event",
+          "spec_url": "https://wicg.github.io/netinfo/#dom-networkinformation-onchange",
+          "support": {
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": {
+              "version_added": "38"
```

Didn't verify if this passes CI

```suggestion
              "version_added": "38"
              "notes": "Removal proposed in <a href=' https://crbug.com/699892'>bug 699892</a>."
```